main/com.tunein/manifest.tt

```diff
@@ -0,0 +1,139 @@
+//    copyright notice, this list of conditions and the following
+//    disclaimer in the documentation and/or other materials
+//    provided with the distribution.
+// 3. Neither the name of the copyright holder nor the names of its
+//    contributors may be used to endorse or promote products derived
+//    from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+// FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+// COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+// INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+// (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+// HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+// OF THE POSSIBILITY OF SUCH DAMAGE.
+
+class @com.tunein
+#_[thingpedia_name="Tunein"]
+#_[thingpedia_description="Listen to internet radio"]
+#_[canonical="radio"]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[subcategory="media"]
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  monitorable query list search_channel(in req query: String
+                                        #_[prompt="Which channel do you want to stream?"] 
+                                        #[string_values="tt:search_query"],
+                                        in opt count: Number 
+                                        #_[prompt="How many search results do you want?"],
+                                        out text: String 
+                                        #[string_values="com.tunein:text"],
+                                        out subtext: String 
+                                        #[string_values="com.tunein:subtext"],
+                                        out current_track: String 
+                                        #[string_values="com.tunein:current_track"],
+                                        out now_playing_id: String 
+                                        #[string_values="com.tunein:now_playing_id"], 
+                                        out URL: Entity(tt:url)
+                                        out formats: String 
+                                        #[string_values="com.tunein:formats"],
+                                        image: Entity(tt:picture))
+  #_[canonical=["search radio channel", "find radio channel"]]
+  #_[confirmation="Radio channels matching $query"]
+  #_[formatted=[{type="rdl",webCallback="${URL}",displayTitle="${text}",displayText="${subtext}"}, {type="picture",url="${image}"}]]
```

Comment:
You can merge the picture and the RDL into one card: RDL objects have a "pictureUrl" property.
(docs: https://almond-static.stanford.edu/jsdoc/thingpedia/latest/interfaces/formatobjects.rdl.html)